src/dmd/declaration.d

```diff
@@ -957,6 +967,13 @@ extern (C++) final class AliasDeclaration : Declaration
         return this;
     }
 
+    /** Returns: `true` if this instance was created to make a template parameter
+    visible in the scope of a tempalte body, `false` otherwise */
+    bool isCreatedForTemplateParameter() const
```

Comment:
isAliasedTemplateParameter?

"Is created for template parameter" is awkward English.

Reply:
yes, i know but the name you suggest might be confusing with `TemplateAliasParameter`, which is totally another thing.

Reply:
no ok finally

Reply:
If you are really concerned, can always keep the original name `wasTemplateParameter()`.  It's not _that_ big a deal at the end, just so long as the name is neither ambiguous nor awkwardly worded.